src/Symfony/Bridge/Monolog/Handler/MailerHandler.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Monolog\Handler;
+
+use Monolog\Formatter\FormatterInterface;
+use Monolog\Formatter\HtmlFormatter;
+use Monolog\Formatter\LineFormatter;
+use Monolog\Handler\MailHandler;
+use Monolog\Logger;
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Mime\Email;
+
+/**
+ * @author Alexander Borisov <boshurik@gmail.com>
+ */
+class MailerHandler extends MailHandler
```

Comment:
I would not extend `MailHandler` here and use `ProcessableHandlerTrait` and `FormattableHandlerTrait` for better compat with Monolog 2.

Reply:
@fabpot what benefits of not extending `MailHandler`? As I can see it is still a part of Monolog 2. There is no notices about it in `CHANGELOG.md`. On the other hand we must reimplement [\Monolog\Handler\MailHandler::handleBatch](https://github.com/Seldaek/monolog/blob/master/src/Monolog/Handler/MailHandler.php#L27) and [\Monolog\Handler\MailHandler::getHighestRecord](https://github.com/Seldaek/monolog/blob/master/src/Monolog/Handler/MailHandler.php#L59) which are basically the only methods in `MailHandler`. Maybe use traits is enough in this case?

Reply:
@fabpot tried existing implementation on monolog 2.0 and it works fine